packages/react/src/themes/teams/siteVariables.ts

```diff
@@ -29,8 +29,7 @@ export const shadowLevel1 = '0 .2rem .4rem -.075rem rgba(0, 0, 0, .1)'
 export const shadowLevel2 = '0 .4rem .7rem -.1rem rgba(0, 0, 0, .1)'
 export const shadowLevel3 = '0 .8rem 1rem -.2rem rgba(0, 0, 0, .1)'
 export const shadowLevel4 = '0 1.6rem 1.8rem -.4rem rgba(0, 0, 0, .1)'
-export const shadowLevel1Dark = '0 .2rem .4rem -.075rem rgba(0, 0, 0, .25)'
-export const shadowLevel1Darker = '0 .2rem .4rem -.075rem rgba(0, 0, 0, .5)'
```

Comment:
This is from the angular app:

.shadow-level-1 {
    box-shadow: 0 .2rem .4rem -.075rem rgba(0,0,0,.1)
}

.shadow-level-1-dark {
    box-shadow: 0 .2rem .4rem -.075rem rgba(0,0,0,.25)
}

.shadow-level-1-darker {
    box-shadow: 0 .2rem .4rem -.075rem rgba(0,0,0,.5)
}

.shadow-level-2 {
    box-shadow: 0 .4rem .7rem -.1rem rgba(0,0,0,.1)
}

.shadow-level-3 {
    box-shadow: 0 .8rem 1rem -.2rem rgba(0,0,0,.1)
}

.shadow-level-4 {
    box-shadow: 0 1.6rem 1.8rem -.4rem rgba(0,0,0,.1)
}

Reply:
where is 'box-shadow: 0 .2rem .4rem -.075rem rgba(0,0,0,.5)' currently used in Angular?

Reply:
only two places: action-marker and zoom-level-indicator for desktop

Reply:
in Action Marker (which I believe is the @mention, important, etc) and the "zoom level indicator desktop" (and I'm not sure what that is exactly)
![image](https://user-images.githubusercontent.com/15043460/68347881-2ba81780-00ad-11ea-9250-2d6d358e1636.png)

Reply:
zoom-level-indicator in n* can move to the new confirmation component (which does not need 0 .2rem .4rem -.075rem rgba(0,0,0,.5). 
![image](https://user-images.githubusercontent.com/48726002/68348185-1d0e3000-00ae-11ea-8502-13c607c749e1.png)

action-marker can move to box-shadow: 0 .2rem .4rem -.075rem rgba(0,0,0,.25) (verified with Rebecca on messaging.)

Reply:
So should we actually keep "Dark" and make primary use the shadow-1-dark since it is aligned with the Angular app? If someone later adds Darker then fine - rather than "shadow-1-darkerer"? :)

Reply:
Sure, I am not particular on naming here. :)

Reply:
I'm just concerned about possible future confusion between Angular and Fluent variable names and how you have it recorded/documented.
How much work is it to update redlines/documentation to refer to 0.25 opacity as "shadow-level-1-dark"?